pkgs/tools/security/pass/default.nix

```diff
@@ -18,8 +19,12 @@ let
     repo   = "pass-${p.name}";
     inherit (p) rev sha256;
   })) [
-    { name = "import"; rev = "491935bd275f29ceac2b876b3a288011d1ce31e7"; sha256 = "02mbh05ab8h7kc30hz718d1d1vkjz43b96c7p0xnd92610d2q66q"; }
-    { name = "update"; rev = "cf576c9036fd18efb9ed29e0e9f811207b556fde"; sha256 = "1hhbrg6a2walrvla6q4cd3pgrqbcrf9brzjkb748735shxfn52hd"; }
+    { name = "import";
+      rev = "491935bd275f29ceac2b876b3a288011d1ce31e7"; sha256 = "02mbh05ab8h7kc30hz718d1d1vkjz43b96c7p0xnd92610d2q66q"; }
+    { name = "update";
+      rev = "cf576c9036fd18efb9ed29e0e9f811207b556fde"; sha256 = "1hhbrg6a2walrvla6q4cd3pgrqbcrf9brzjkb748735shxfn52hd"; }
+    { name = "tomb";
+      rev = "3368134898a42c1b758fabac625ec240e125c6be"; sha256 = "0qqmxfg4w3r088qhlkhs44036mya82vjflsjjhw2hk8y0wd2i6ds"; }
```

Comment:
Do all you users want these plugins?

Reply:
I agree that all those plugins will probably not be used by everyone but, as there is already 2 plugins in the package, I thought that adding a third one will not hurt. Also, correct me if I am wrong, but it seems that it is not possible to add plugin afterwards, so it has to be in the pass package.

Maybe we can add an option to the package for the the user to selected the wanted plugins?

Reply:
I think they were just added by default for simplicity. But now as we get plugins with additional dependencies, it starts to make sense to rework this part.

Reply:
The proper way to do this is to add a wrapper (like vim) that then pulls in all the required runtimes.